File: The Core/Intro Gates/7. Late Ride/lateRide.js
function lateRide(n) {
    var hour = Math.floor(n / 60);
    var minutes = n % 60;
    
    var hoursSum = Math.floor(hour / 10) + hour % 10;
    var minutesSum = Math.floor(minutes / 10) + minutes % 10;
    
    return hoursSum + minutesSum;    
}

File: The Core/Intro Gates/7. Late Ride/lateRide.lua
function lateRide(n)
    hour = math.floor(n / 60)
    minutes = n % 60
    
    hoursSum = math.floor(hour / 10) + hour % 10
    minutesSum = math.floor(minutes / 10) + minutes % 10

    return hoursSum + minutesSum
end
